upstairs/src/upstairs.rs

```diff
@@ -1757,18 +1718,14 @@ impl Upstairs {
                         // Copy the region definition into the Downstairs
                         self.downstairs.set_ddef(self.ddef.get_def().unwrap());
                         // See if we have a quorum
-                        if self.connect_region_set() {
-                            // We connected normally, so there's no need
-                            // to check for live-repair.
-                            self.repair_check_deadline = None;
-                        }
+                        self.connect_region_set();
                     }
                     Ok(NegotiationResult::Replay) => {
                         self.downstairs.replay_jobs(client_id);
                     }
                     Ok(NegotiationResult::LiveRepair) => {
-                        // Immediately check for live-repair
-                        self.repair_check_deadline = Some(Instant::now());
+                        // We will immediately check for live-repair as part of
```

Comment:
Would this place be good to log a message that someone wants live repair?
Meaning, we could get one message and one message only?  Verify range to target?

Reply:
Good idea, done in de089ab41583